mob.go

```diff
@@ -264,38 +267,46 @@ func currentCliName(argZero string) string {
 func getDefaultConfiguration() Configuration {
 	voiceCommand := ""
 	notifyCommand := ""
+	openCommand := ""
 	switch runtime.GOOS {
 	case "darwin":
 		voiceCommand = "say \"%s\""
 		notifyCommand = "/usr/bin/osascript -e 'display notification \"%s\"'"
+		openCommand = "open -e %s"
 	case "linux":
 		voiceCommand = "say \"%s\""
 		notifyCommand = "notify-send \"%s\""
+		openCommand = "editor %s"
 	case "windows":
 		voiceCommand = "(New-Object -ComObject SAPI.SPVoice).Speak(\\\"%s\\\")"
+		openCommand = "Notepad \"%s\""
```

Comment:
I don't think default values for this open command make sense. This is something everybody has to set explicitly themselves.

Reply:
I also thought about this. I agree we should check if this command is set and if not we should show a warning.

Reply:
done